packages/react-core/src/components/ToggleGroup/examples/ToggleGroupDefaultMultiple.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { ToggleGroup, ToggleGroupItem, Button, Stack, StackItem } from '@patternfly/react-core';
+
+export const ToggleGroupDefaultMultiple: React.FunctionComponent = () => {
+  const [state, setState] = React.useState({
+    isSelected: {
+      first: false,
+      second: false
```

Comment:
Similar to my suggestion in the compact example, let's update these keys (and their respective `buttonId` props) to be a bit more descriptive:

```suggestion
      'toggle-group-multiple-1': false,
      'toggle-group-multiple-2': false
```

This will help future proof things by avoiding potentially duplicate ID's down the line. This would also apply to all of the example files.